### PR TITLE
separate mouse sensitivities for turn/strafe/move/look

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

File: src/m_misc.c [hunk 1]
```diff
@@ -493,14 +493,28 @@ default_t defaults[] = {
...
     (config_t *) &mouseSensitivity_horiz, NULL,
     {10}, {0,UL}, number, ss_none, wad_no,
-    "adjust horizontal (x) mouse sensitivity"
+    "adjust horizontal (x) mouse sensitivity for turning"
   },
 
   { //jff 4/3/98 allow unlimited sensitivity
     "mouse_sensitivity_vert",
     (config_t *) &mouseSensitivity_vert, NULL,
     {10}, {0,UL}, number, ss_none, wad_no,
```
Comment (rfomin) [inline src/m_misc.c:502]: I think we should set `mouse_sensitivity_vert` and `mouse_sensitivity_horiz_strafe` to `0` by default. We also have a vertmouse toggle which is disabled by default.

**Comment by rfomin** [comment]
Hmm something wrong with graphics lumps?
woof0000

File: src/m_menu.c [hunk 2]
```diff
@@ -1467,9 +1475,13 @@ enum
 menuitem_t MouseMenu[]=
 {
   // [FG] alternative text for missing menu graphics lumps
-  {2,"M_HORSEN",M_MouseHoriz,'h', "HORIZONTAL"},
+  {2,"M_HORSEN",M_MouseHoriz,'h', "HORIZONTAL: TURN"},
+  {-1,"",0},
+  {2,"M_HORSEN2",M_MouseHoriz2,'h', "HORIZONTAL: STRAFE"},
```
Comment (rfomin) [inline src/m_menu.c:1480]: `M_HORSEN2` is 9 characters long, so it is truncated to `M_HORSEN`.

Comment (fabiangreffrath) [reply src/m_menu.c:1480]: LOL! What do you suggest?

Comment (rfomin) [reply src/m_menu.c:1480]: Alternatively, we can move the mouse sensitivity to the General menu and keep the vanilla look of the Mouse Sensitivity menu. We can use the mini thermo in the General menu.

Comment (fabiangreffrath) [reply src/m_menu.c:1480]: He, but we have a "Mouse Sensitivity" menu for this exact reason. I'd be surprised to find one specific mouse sensitivity setting elsewhere. Also, even if we move just one vertical mouse sensitivity setting out of the way, we'd still end up with two "Horizontal" settings and thus two `M_HORSEN` patches.

I am inclined to do just this and say "goodbye" to patches drawing in the "Mouse Sensitivity" menu:

```patch
--- a/src/m_menu.c
+++ b/src/m_menu.c
@@ -1475,13 +1475,13 @@ enum
 menuitem_t MouseMenu[]=
 {
   // [FG] alternative text for missing menu graphics lumps
-  {2,"M_HORSEN",M_MouseHoriz,'h', "HORIZONTAL: TURN"},
+  {2,"",M_MouseHoriz,'h', "HORIZONTAL: TURN"},
   {-1,"",0},
-  {2,"M_HORSEN2",M_MouseHoriz2,'h', "HORIZONTAL: STRAFE"},
+  {2,"",M_MouseHoriz2,'h', "HORIZONTAL: STRAFE"},
   {-1,"",0},
-  {2,"M_VERSEN",M_MouseVert,'v', "VERTICAL: MOVE"},
+  {2,"",M_MouseVert,'v', "VERTICAL: MOVE"},
   {-1,"",0},
-  {2,"M_VERSEN2",M_MouseVert2,'v', "VERTICAL: LOOK"},
+  {2,"",M_MouseVert2,'v', "VERTICAL: LOOK"},
   {-1,"",0}
 };

```

Comment (rfomin) [reply src/m_menu.c:1480]: Sure, I agree with Crispy's approach.

**Review by rfomin — APPROVED** [review]

**Comment by fabiangreffrath** [comment]
Thank you for your input!

**Pull request merged**